scripts/extensions/datetimeField/src/extension.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+
+import {
+    ISuperdesk,
+    IExtension,
+    IExtensionActivationResult,
+    IPreviewComponentProps,
+} from 'superdesk-api';
+import {getDateTimeField} from './getDateTimeField';
+import {getConfigComponent} from './getConfigComponent';
+
+class DateTimePreview extends React.PureComponent<IPreviewComponentProps> {
+    render() {
+        if (this.props.value == null) {
+            return null;
+        } else {
+            return <div>{new Date(this.props.value).toLocaleString()}</div>;
```

Comment:
imo we should provide helpers for datetime formatting via ISuperdesk, we have configs for that so we should keep it consistent